geometry3d/graph2vol.cpp

```diff
@@ -0,0 +1,224 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+/**
+ * @file
+ * @ingroup geometry3d
+ * @author Bertrand Kerautret (\c bertrand.kerautret@univ-lyon2.fr )
+ * 
+ *
+ * @date 2023/04/20
+ *
+ * Source file of the tool graph2vol
+ *
+ * This file is part of the DGtal library/DGtalTools-contrib Project.
+ */
+
+///////////////////////////////////////////////////////////////////////////////
+#include "DGtal/base/Common.h"
+#include "DGtal/helpers/StdDefs.h"
+#include <DGtal/io/readers/TableReader.h>
+#include <DGtal/io/readers/PointListReader.h>
+#include "DGtal/io/writers/GenericWriter.h"
+
+#include "CLI11.hpp"
+
+///////////////////////////////////////////////////////////////////////////////
+using namespace std;
+using namespace DGtal;
+///////////////////////////////////////////////////////////////////////////////
+
+
+/**
+ @page graph2vol graph2vol
+ 
+ @brief  Description of the tool...
+
+ @b Usage:   graph2vol [input]
+
+ @b Allowed @b options @b are :
+ 
+
+ @code
+ -h,--help                             Print this help message and exit
+  -v,--inputVertex TEXT:FILE REQUIRED   input file containing the vertex list.
```

Comment:
So the vertex list is a sequence of points in Z³ ?

Reply:
If so, it should be mentioned that the "embedding" of the vertices is explicitly given